API.pm

```diff
@@ -469,10 +471,6 @@ sub getPlaylistTracks {
 		extra       => 'tracks',
 		limit       => QOBUZ_USERDATA_LIMIT,
 		_extractor  => 'tracks',
-		_maxKey     => sub {
-			my ($results) = @_;
-			$results->{tracks_count};
-		},
```

Comment:
Looking at the changes in `paginGet` I believe it would have been simpler to change this from `$results->{tracks_count}` to `$results->{tracks}->{count}`? What am I missing?

Reply:
Why do you want to read out 'track_count' when it is clear that 'total' has to contain the same value? Or am I missing something.?
-----------------
Warum willst Du 'track_count' auslesen wenn doch klar ist , dass in 'total' der gleiche Wert drin stehen muss? Oder übersehe ich da etwas.?

Reply:
`_maxKey` wurde eingeführt, um das Maximum zu eruieren. Wenn das im bestehenden Code nicht korrekt ist, dann müssen wir diese Funktion flicken, nicht irgendwo anders den Code so ändern, dass auch der `_extractor` hierfür "missbraucht" werden kann. Es gibt zwei Funktionen für unterschiedliche Dinge. Wir sollten die nicht mischen.

Reply:
Nicht korrekt ist es nicht, aber man braucht es an der Stelle nicht, weil die Standardfunktion für _maxKey in _pagingGet() bereits den richtigen Wert liefert.

my $getMaxFn = ref $params->{_maxKey} ? delete $params->{_maxKey} : sub {
		my ($results) = @_;
		$results->{$params->{_maxKey}}->{total};
	};

Mir fällt noch was ein, morgen mehr, muss leider weg, ein schönes Wochenende.